### ISSUE TITLE
Make Atlantean more sea-based?

### ISSUE THREAD

**Issue by tribnia** [DominiqueMakowski/AntediluvianBlood#1]
e.g. give them a rather weak army, but make it stronger on coastal provinces, and cheaper to get on board. 
Give them the CB to conquest coastal county, like Norse in CK2 do.
No longer _Sanctity of Nature_, since they should have a closer connection with the sea than the forest.
_Sun Worship_ tenet for Atlantean could be adjusted to be more suitable for someone under the sea and do not see the sun quite often. As well as _Astrology_.
Their dna... could be like Greek, but brighter, or darker. Brighter if they spend more time under water, darker if spend more time as sailors. Maybe two separate traits could be made to show these difference?

**Comment by DominiqueMakowski** [comment]
Yes that all makes sense for the Antlanteans *per se*, though first we'd have to reorganize the culture group and the subcultures. I don't think it makes sense that all of them are atlanteans currently, best would be to have one group (the antediluvians), and then one subculture for each starting point. The Atlanteans would be the most suited for the group in the Canaries islands (the closest to the supposed location in the Atlantic beyond Gibraltar). 

Since Plato suggests that his account of Atlantis derives from Egyptian priests, maybe they could have a mix of Greek and "Egyptian gods"-like names, which could nicely fit with their southern location.

> Sun Worship tenet for Atlantean could be adjusted

Yes, having "Sea Worship" would be great, but it requires designing/adapting an image for the tenet...

**Comment by tribnia** [comment]
![f603918fa0ec08fabd3c1e2e55ee3d6d55fbda7f](https://user-images.githubusercontent.com/40262618/99896706-08f39b80-2cce-11eb-82f5-c31c8f653e39.jpg)
What about this great artwork by Katsushika Hokusai?
